packages/common/src/utils.ts

```diff
@@ -331,16 +329,11 @@ export function getGlobalObjects<K extends Extract<keyof Window, string>>(
   return result;
 }
 
-export type IntegerType = number | string | bigint | Uint8Array | BN;
+export type IntegerType = number | string | bigint | Uint8Array;
```

Comment:
Seems like it would be pretty easy to maintain bn.js support by creating an `interface` that matched bn.js class instances, like this handful of properties:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ea0e0de908b17dd697e373da9cd0fca8d7e41c66/types/bn.js/index.d.ts#L112-L145

Then copy over a bit of code from `BN.isBN` to use in `intToBigInt(...)`, like this line: https://github.com/indutny/bn.js/blob/db57519421f0c47c9f68c05fa6fc12273dcca2c2/lib/bn.js#L66-L67

That would avoid creating a lot of breaking changes for all the projects still passing in a bn.js instance to various functions.

Reply:
Done!